javacord-api/src/main/java/org/javacord/api/interaction/InteractionBase.java

```diff
@@ -53,15 +53,15 @@
      * Please note that you can only actually update this loading state message within 15 minutes after receiving the
      * interaction.
      *
-     * @param ephemeral wether or not the response should be ephemeral
+     * @param ephemeral Whether the response should be ephemeral
      * @return A CompletableFuture that completes as soon as the acknowledgement has been sent; it yields an updater
      *     that should be used to update the message later on.
      */
     CompletableFuture<InteractionOriginalResponseUpdater> respondLater(boolean ephemeral);
 
     /**
-     * Create a message builder to send follow up messages for this interaction.
-     * You can send, edit and delete follow up messages up to 15 minutes after you received the interaction.
+     * Create a message builder to send follow-up messages for this interaction.
+     * You can send, edit and delete follow-up messages to 15 minutes after you received the interaction.
```

Comment:
Second 'up' was necessary.

```suggestion
     * You can send, edit and delete follow-up messages for up to 15 minutes after you received the interaction.
```